### PR TITLE
Update 5.2.0.md

### PR DESCRIPTION
**Pull request by alexanderkyte**

### CODE REVIEW

File: docs/about-mono/releases/5.2.0.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ To debug the assembly loading process, set the logging environment variables to
 
 The current default behavior is the same as in previous Mono versions. It is explicitly available with the `--assembly-loader=legacy` option.
 
+Mangler V1
+---------------------
+
+An initial implementation of the name mangler was created. It associates with each method a unique string that captures the unique attributes of the method. If two methods have the same mangled representation, they're the same method.
```
Comment (akoeplinger) [inline docs/about-mono/releases/5.2.0.md]: I don't think a typical Mono user knows what the "mangler" is after reading this. Can you add some more details about why/where it is needed and when/how someone might want to use it?

File: docs/about-mono/releases/5.2.0.md [hunk 2]
```diff
@@ -27,6 +27,16 @@ To debug the assembly loading process, set the logging environment variables to
...
+
+Safer Getenv 
+---------------------
+
+The g_getenv and g_setenv are neither threadsafe nor reentrant. Subsequent calls to g_getenv can cause previous strings returned from it to point to garbage memory. This was leading to issues with System.Configuration that could be seen with HttpClient. Now we lock around these calls and return duplicated memory. This is a serious API change to g_getenv, as the returned memory must now be freed.
```
Comment (akoeplinger) [inline docs/about-mono/releases/5.2.0.md]: with `serious API change` do you mean for embedders? this should have more details about what to change then.

**Review by akoeplinger — CHANGES_REQUESTED** [review]
also needs a rebase :)

**Comment by alexanderkyte** [comment]
Comments addressed

**Pull request merged**